apps/studio/src/pages/sites/[siteId]/folders/[folderId]/index.tsx

```diff
@@ -109,29 +109,40 @@ const FolderPage: NextPageWithLayout = () => {
     <>
       <VStack w="100%" p="1.75rem" gap="1rem">
         <VStack w="100%" align="start">
-          <Breadcrumb size="sm">
+          <Breadcrumb size="sm" w="100%">
```

Comment:
should it be 100% though? Never seen such a long breadcrumb either

i thought we could do something like:

folder123 > superlooooo .... oooonnnng > anotherrrrr...rrrlongone >

basically should each section have a max length

Cos in this case if the first one is super long, then it itself will take up the whole. space right?

Reply:
since our designer is ok with this current implementation, should we merge this first and file a follow up issue? i don't really have an issue iwth hte breadcrumb taking up the whole area because that's what the whitespace is for. 

i made the choice to truncate on the last breadcrumb because it's the one that has redundant information - the title already show the information on the breadcrumb and the breadcrumb is also non-clickable so it doesn't contribute anything. 

cc @sehyunidaaa should the max length of hte breadcrumbs be constrained perhaps to `50%`? (put another way, should the breadcrumb be allowed to take the full whitespace)

Reply:
@seaerchin I am good to merge and file a follow up issue!